docker/Dockerfile.l4t-36.4.0

```diff
@@ -6,7 +6,8 @@ RUN apt-get update && apt-get install -y \
   wget \
   && rm -rf /var/lib/apt/lists/*
 
-RUN git clone --branch scarthgap https://github.com/OE4T/meta-tegra.git meta-tegra-scarthgap
+RUN git clone --branch scarthgap https://github.com/OE4T/meta-tegra.git meta-tegra-scarthgap && cd meta-tegra-scarthgap && git checkout 6d1dc9997b4d83889369abf452d07e3f61091e1b
```

Comment:
That commit hash doesn't look correct. The last commit in the scarthgap branch before the R36.4.3 update commits started going in is `e1c238e20c83821b47547cf5956914f43007471e`. The one you have there is from the master branch.